src/resourceGathering/ResourceGatheringBuilder.java

```diff
@@ -50,13 +51,19 @@ public Context build(Context<Object> context) {
 		
 		int resourceCount = (Integer)params.getValue("resource_count");
 		
+		
 		HQ = new Headquarters(space, grid);
 		context.add(HQ);
+		space.moveTo(HQ, space.getDimensions().getHeight()/2, space.getDimensions().getWidth()/2);
 		
 		for (int i = 0; i < robotCount; i++) {
-			context.add(new Robot(space, grid, HQ, maxFuelLevel, fuelRate, maxSensorRange, maxCommunicationRange, i));
+			final Robot robot = new Robot(space, grid, HQ, maxFuelLevel, fuelRate, maxSensorRange, maxCommunicationRange, i);
+			context.add(robot);
+			space.moveTo(robot, space.getLocation(HQ).getX(), space.getLocation(HQ).getY());
```

Comment:
because it looks like you're manually moving all the robots after adding them anyways